athf/agents/base.py

```
@@ -229,17 +229,23 @@ def _log_llm_metrics(
+        try:
+            from athf.metrics import record_llm_call
+
+            record_llm_call(
+                model=model_id,
+                input_tokens=input_tokens,
+                output_tokens=output_tokens,
+                duration_ms=duration_ms,
+                agent=agent_name,
+                cost_usd=cost_usd,
+            )
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)